### PR TITLE
Develop it

### PR DESCRIPTION
**Pull request by aborruso**
It's the italian version

### CODE REVIEW

File: input/cheat-sheet.yaml [hunk 1]
```diff
@@ -1,531 +1,645 @@
...
     desc:
       en: '2019-07-07'
       es: '2019-07-07'
+      it: '2019-07-07'
   - cmd:
       en: More info
       es: Más info
+      it: Maggiori informazioni
     desc:
       en: <a href="https://visidata.org/">visidata.org</a><br/><a href="https://jsvine.github.io/intro-to-visidata/">jsvine.github.io/intro-to-visidata</a>
```
Comment (jsvine) [inline input/cheat-sheet.yaml]: @aborruso Do you want to add a link to your *guida italiana* here? If so, put it as the second link, right after the official VisiData homepage, separated by `<br/>`

File: input/cheat-sheet.yaml [hunk 2]
```diff
@@ -1,531 +1,645 @@
...
       es: Actualizado
+      it: Aggiornato
     desc:
       en: '2019-07-07'
       es: '2019-07-07'
+      it: '2019-07-07'
   - cmd:
       en: More info
       es: Más info
+      it: Maggiori informazioni
```
Comment (jsvine) [inline input/cheat-sheet.yaml]: @aborruso Would "Maggiori info" be understood? (Just to keep things *un po 'corto*.)

**Comment by jsvine** [comment]
A big thanks, @aborruso! Very exciting! I added a couple of comments/questions for you. Once those are resolved, I'll merge this into the `develop-it` branch, and run the build process. If the new text can't quite fit on a single-page PDF, I'll suggest some translations that we might try shortening.

**Comment by aborruso** [comment]
Dear @jsvine I have read and applied your notes

**Comment by jsvine** [comment]
Great! It looks like the translations are slightly long; they don't all fit on one page. I'm attaching the rendered PDF. By comparing it to the English version, perhaps you can see some translations that can be shortened?

[visidata-cheat-sheet.it.pdf](https://github.com/jsvine/visidata-cheat-sheet/files/3373101/visidata-cheat-sheet.it.pdf)

**Comment by aborruso** [comment]
I will work on it as soon as possible.

Thank you very much

______
Scusami per la brevità, ti sto scrivendo dal cellulare.

website: https://medium.com/tantotanto
38° 7' 48" N, 13° 21' 9" E EPSG:4326

Il mar 9 lug 2019, 15:13 Jeremy Singer-Vine <notifications@github.com> ha
scritto:

**Comment by aborruso** [comment]
I have created a shorter one https://github.com/jsvine/visidata-cheat-sheet/pull/4

**Pull request merged**